test/a11y.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+const async = require('async');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).